eks-distro-base/Makefile

```diff
@@ -240,6 +258,77 @@ define CREATE_PR_TARGET_BODY
 	)
 endef
 
+# $1 - language short name (python/nodejs/golang/java)
```

Comment:
The following blob is the real meat of this PR, its very magicial. I tried to document where it made sense, please ask if anything isnt clear and I can improve the comments/code.

Reply:
# 🪄